src/documentation/sdks/nexmark.md

```diff
@@ -112,17 +122,19 @@ Performance:
 
 ## Benchmark launch configuration
 
-We can specify the Beam runner to use with maven profiles, available profiles are:
-
-    direct-runner
-    spark-runner
-    flink-runner
-    apex-runner
+The Nexmark launcher accepts the `--runner` argument as usual for programs that
+use Beam PipelineOptions to manage their command line arguments. In addition
+to this, the necessary dependencies must be configured.
 
-The runner must also be specified like in any other Beam pipeline using:
+When running via Gradle, the following two parameters control the execution:
 
-    --runner
+    -P nexmark.args
+        The command line to pass to the Nexmark main program.
 
+    -P nexmark.runner
```

Comment:
no more profiles for the runners then? The needed dependencies are all included in the runner projet specified by the user?

Reply:
Yes, Gradle puts them on the classpath. The one exception is the Spark runner. I used to have a `-P nexmark.provideSpark` parameter to keep it a little decoupled, but since it is pretty much still "magic" knowledge, I followed @lukecwik's suggestion and just hardcoded knowledge of the dependency, so I did that [here](https://github.com/apache/beam/blob/master/sdks/java/nexmark/build.gradle#L35).

I think when I add the fat jar capability it may require a separate fat jar for each known runner config. At that point, pretending that the runner dependency can be anything (versus a selection of a hardcoded profile) may not be useful.

Reply:
The alternative which I mean is that we could just make it like before where you choose a profile and `-P nexmark.runner` is a choice from an enum, not a magic Gradle subproject name.

Reply:
Yes I think I prefer this explicit version rather than the magic one.

Reply:
OK. Filed https://issues.apache.org/jira/browse/BEAM-4139 in the gradle improvements to go back to the repo and change it.